Formula/gnuplot.rb

```diff
@@ -47,23 +48,27 @@ class Gnuplot < Formula
   depends_on "webp"
   depends_on "pango" if build.with?("cairo") || build.with?("wxmac")
   depends_on "pdflib-lite" => :optional
-  depends_on "qt" => :optional
+  depends_on "qt5" => :optional
   depends_on "wxmac" => :optional
   depends_on :tex => :optional
   depends_on :x11 => :optional
 
   def install
-    if build.with? "aquaterm"
+    if build.with?("aquaterm")
       # Add "/Library/Frameworks" to the default framework search path, so that an
       # installed AquaTerm framework can be found. Brew does not add this path
       # when building against an SDK (Nov 2013).
       ENV.prepend "CPPFLAGS", "-F/Library/Frameworks"
       ENV.prepend "LDFLAGS", "-F/Library/Frameworks"
     end
 
+    if build.with?("qt5")
+      # make sure qt5 is found by configure
+      ENV.prepend_path "PATH", Formula["qt5"].bin
```

Comment:
You are right. However, this is strange: I was playing with the formula for Octave 4.2 (which is the reason that I am interested in qt5) and it was always using the wrong moc if the path was not adapted.